cloud_service/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -27,39 +27,76 @@ The following sections provide an overview of installation steps and links to th
 [id="securing-rh-cloud-clusters"]
 === Securing {osp} clusters
 
-To secure {osp} clusters by using the Operator, perform the following steps:
+You can secure {osp} clusters by using the Operator, Helm charts, or the `roxctl` CLI.
```

Comment:
Do we need to state which Operator, as this is the first reference in the module?